src/pombot/index.js

```diff
@@ -11,16 +11,19 @@ import statusCommand from './commands/status';
 // create bot
 const bot = createSlackBot({
   name: 'Pombot',
-  slack: {
-    rtmClient: new RtmClient(config.tokens.pombot, {
-      dataStore: new MemoryDataStore(),
-      autoReconnect: true,
-    }),
-    webClient: new WebClient(config.tokens.pombot),
+  getSlack() {
```

Comment:
I just didn't want to create the slack instance until the `login` method was called, in case it was never called because the token wasn't available.

If the IRC bot needs an instance of `IRC`, I'd imagine I'd do it the same way.